README.rst

```diff
@@ -96,12 +96,13 @@ Integration tests can be executed on a remote server.
 
 .. code-block:: shell
 
-    docker-compose run remotesettings:tests \
-        -e SERVER=http://settings.dev.mozaws.net \
-        -e MAIL_DIR="" \  # disable tests about emails.
-        -e SKIP_SERVER_SETUP=true \
-        -e EDITOR_AUTH=editor:azerty123" \
-        -e REVIEWER_AUTH=reviwer:s3cr3t"
+    docker-compose run \
+    --env SERVER=https://settings.dev.mozaws.net/v1 \
+    --env MAIL_DIR="" \
```

Comment:
Yep. I removed it because it because without the comment you can copy+paste from the docs, but I'll use [this method](https://stackoverflow.com/a/1456019) to include the comment